subsys/bluetooth/crypto/Kconfig

```diff
@@ -2,9 +2,23 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BT_CRYPTO
+	bool "Select crypto backend for Bluetooth Cryptographic Toolbox"
```

Comment:
`BT_CRYPTO` used to be promptless, is there a reason to give it a prompt?

Reply:
Right, this is a leftover from a change I tried during the development. I will revert it